.circleci/docker/install_valgrind.sh

```diff
@@ -9,7 +9,7 @@ tar -xjf valgrind-${VALGRIND_VERSION}.tar.bz2
 cd valgrind-${VALGRIND_VERSION}
 ./configure --prefix=/usr/local
 make -j6
-sudo make install
```

Comment:
I assume the reason why we can drop "sudo" is because we run the docker with "--privileged"?

Reply:
I found the script is executed by `root` in dockerfile, which is already in sudo